src/Servant/Swagger/Internal.hs

```diff
@@ -225,6 +225,10 @@ instance (KnownSymbol sym, ToParamSchema a, HasSwagger sub) => HasSwagger (Captu
             & in_ .~ ParamPath
             & paramSchema .~ toParamSchema (Proxy :: Proxy a))
 
+-- | Swagger Spec doesn't have a notion of CaptureAll, this instance is the best effort.
+instance (KnownSymbol sym, ToParamSchema a, HasSwagger sub) => HasSwagger (CaptureAll sym a :> sub) where
```

Comment:
ping @fizruk, I'm not sure about this.

One solution would be to not define this instance at all. OTOH we need it for comprehensiveAPI.